README.md

```diff
@@ -18,12 +66,54 @@ to Moodle courses and the API from tool_opencast to load the video metadata.
 
 When a video is set to OER it also stores changes in metadata back to the video (license).
 
-## On OER release
+### On OER release
 
 The released videos remain in Opencast and the link is provided in the release metadata.
 The video will be set to public.
 
-## TODOs / Known issues
+## Configuration
+
+The following setting is available (by default enabled):
+
+- Add the people and their roles automatically to the OER element when loading the videos from opencast. People will only be added when the element metadata has not been edited and stored. More people can be added manually and also the automatically added people can be removed.
+Following roles will be added: Presenter, Contributor, Rightsholder.
+
+## Usage
+
+See description of main plugin [local_oer](https://github.com/llttugraz/moodle-local_oer).
+
+## Dependencies
+
+* [local_oer](https://github.com/llttugraz/moodle-local_oer) v2.3.0+
+* [tool_opencast](https://moodle.org/plugins/tool_opencast) v4.3-r1+
+* [block_opencast](https://moodle.org/plugins/block_opencast) v4.3-r1+
+
+## API Documentation
+
+No API.
+
+## Subplugins
+
+No subplugins.
+
+## Privacy
+
+No personal data are stored.
+
+## Known Issues
 
 * Deleting of videos cannot be prevented yet.
-* Support for multiple OpenCast instances, connected to one Moodle, has to be added
\ No newline at end of file
+* Support for multiple OpenCast instances, connected to one Moodle, has to be added
```

Comment:
Support for multiple Opencast instances, connected to one Moodle instance, has to been implemented yet.